packages/tools/src/npm/index.coffee.md

```diff
@@ -43,9 +43,12 @@ console.info(`Package was installed: ${status}`)
           $ref: 'module://@nikitajs/engine/src/actions/execute#/properties/sudo'
         'upgrade':
           type: 'boolean'
-          default: false
           description: 'Upgrade all packages.'
-      required: ['name']
+      oneOf: [
+        required: ['name']
+      ,
+        required: ['upgrade']
+      ]
```

Comment:
not your fault but this is wrong. The `tools.npm` action shall install and upgrade a single package. To upgrade all packages, it shall be another action `tools.npm.upgrade`. Sorry about that, the description was a mistake, the tests seems ok. The source of truth is not the doc, they are the tests.

Reply:
done